modules/notification/webhook/webhook.go

```diff
@@ -182,7 +182,7 @@ func (m *webhookNotifier) NotifyIssueChangeTitle(doer *models.User, issue *model
 		err = webhook_module.PrepareWebhooks(issue.Repo, models.HookEventPullRequest, &api.PullRequestPayload{
 			Action: api.HookIssueEdited,
 			Index:  issue.Index,
-			Changes: &api.ChangesPayload{
+			Changes: &api.PullRequestChangesPayload{
```

Comment:
In fact, issue also has a branch name. So maybe it's unnecessary to use a seperate struct for issue and pull request?

Reply:
Why would Issue have a branch name? I don't see one in the model.

Reply:
> 
> 
> Why would Issue have a branch name? I don't see one in the model.

@saitho It's `Issue.Ref`.

Reply:
Oh! I didn't know that you could assign an issue to a branch.
I added `Ref` to ChangesPayload and adjusted the usages.